src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -164,7 +148,7 @@ fun toHardstop(
   licenceVersion = licence.licenceVersion,
   earliestReleaseDate = earliestReleaseDate,
   isEligibleForEarlyRelease = isEligibleForEarlyRelease,
-  isInHardStopPeriod = licence.isInHardStopPeriod(),
+  isInHardStopPeriod = isInHardStopPeriod,
```

Comment:
not super convinced we need this? wouldn't this always be true?

Reply:
Mentioned in Slack too but also putting here for visibility, would we not want it to be `false` after release, at which point we can't default to `true` as there will be a period between release and PP review?